planet/clients/orders.py

```diff
@@ -313,6 +369,17 @@ async def download_order(self,
                                       progress_bar=progress_bar)
             for location in locations
         ]
+        if checksum:
+            # Checksum Implementation
+            # Get manifest filepath
+            manifest_json = ' '.join(
+                [x for x in filenames if x.endswith('manifest.json')])
+            # Open manifest file and pass to checksum function
```

Comment:
Just a little nit: you could delete these comments because the code is already very clear. Comments are good when needed but can be distracting when not needed.